system/Config/Config.php

```diff
@@ -70,7 +70,7 @@ public static function get(string $name, bool $getShared = true)
 			$class = substr($name, $pos + 1);
 		}
 
-		$class = strtolower($class);
+		//$class = strtolower($class);
```

Comment:
Please remove the line instead of commenting it out. If we need something back, we can always look back through the history. Thanks.